### PR TITLE
few small changes to make the common tests truly common

### PR DESCRIPTION
**Pull request by Plamen5kov**

### CODE REVIEW

**Comment by Plamen5kov** [comment]
ping: @jasssonpet  @fealebenpae

File: Require/index.js [hunk 1]
```diff
@@ -209,6 +209,6 @@ describe("TNS require", function () {
 
     it("should handle JSON file errors", function () {
         require('./RequireJsonCorruptFile');
-        expect(TNSGetOutput()).toBe("File: 'file:///app/shared/Require/RequireJsonCorruptFile/test.json'. JSON Parse error: Unterminated string");
+        expect(TNSGetOutput()).toBe("There was a corrupt or invalid package.json.");
```
Comment (jasssonpet) [inline Require/index.js]: We try to keep the original error when parsing JSON files. I wonder how can we make this somehow common. What does the V8 engine throw as an error message?

File: Require/RequireJsonCorruptFile/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 try {
     require('./test.json');
 } catch (e) {
-    TNSLog(e.message);
+	TNSLog("There was a corrupt or invalid package.json.");
```
Comment (jasssonpet) [inline Require/RequireJsonCorruptFile/index.js]: indentation

Comment (Plamen5kov) [inline Require/RequireJsonCorruptFile/index.js]: although it looks wierd here it's one tab, it should look fine when you download it localy

Comment (jasssonpet) [inline Require/RequireJsonCorruptFile/index.js]: Tab size is relative to the text editor.

File: Require/index.js (hunk 1 above)
Comment (Plamen5kov) [inline Require/index.js]: in this particular case e.message is `Unexpected token`

File: Require/RequireJsonCorruptFile/index.js (hunk 2 above)
Comment (Plamen5kov) [inline Require/RequireJsonCorruptFile/index.js]: well it's 4 in my case :D

File: Require/index.js (hunk 1 above)
Comment (jasssonpet) [inline Require/index.js]: Something like `.toMatch(/JSON Parse error: Unterminated string$|Uncaught SyntaxError: Unexpected token ILLEGAL$/)` looks good to me. Would be glad to hear other opinions on this.

Comment (Plamen5kov) [inline Require/index.js]: tested with: `.toMatch(/JSON Parse error: Unterminated string$|Unexpected token/)` and it works for android when the TnsLog is reverted to the way it was before -> `TNSLog(e.message);`

**Comment by jasssonpet** [comment]
:+1:

**Pull request merged**